runtimes/protocol/identity-management.ts

```diff
@@ -0,0 +1,22 @@
+import { ProtocolRequestType } from './lsp'
+
+export type ListProfilesParams = {}
+export type ListProfilesResult = {
+    profiles?: any[]
+    ssoSessions?: any[]
```

Comment:
Is there now way to scope this down from `any`?

Reply:
Below is the original Toolkits design from the TKC: Auth Design doc.  It only takes into account non-legacy SSO profiles/sso-sessions currently, but there is an expectation that it will be extended to support other profile types in the future.  

There are quite a few fields and profile types which would be time-consuming and unnecessary to build out upfront.  I originally envisioned having separate TypeScript types and JSON-RPC params for each profile type to make it easier for consuming code to work with rather than a single profile type with every possible field on it.  The latter would push validation and which fields apply logic into all of the consumers that would likely be implemented inconsistently and potentially drift over time.

The AWS CLI docs probably have the most complete definition of the possible fields.  https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html

```
export type SsoTokenProfileKind = 'SsoToken'

export type ProfileKind = SsoTokenProfileKind

export const ProfileKind = {
    SsoToken: 'SsoToken'
} as const

export interface Section {
    name: string
}

export interface Profile extends Section {
    readonly kind: ProfileKind
    region?: string
}

export interface SsoTokenProfile extends Profile {
    readonly kind: SsoTokenProfileKind
    ssoSessionName: string
}

export interface SsoSession extends Section {
    ssoStartUrl: string
    ssoRegion: string
    ssoRegistrationScopes?: string[]
}

export interface ListProfilesResult {
    profiles?: (Profile | SsoTokenProfile)[]
    ssoSessions?: SsoSession[]
}
```